draft-ietf-quic-multipath.md

```diff
@@ -885,6 +884,37 @@ two or more active paths during the connection lifetime. Different applications
 Once the implementation has decided which paths to keep alive, it can do so by sending Ping frames
 on each of these paths before the idle timeout expires.
 
+## Connection ID Changes and NAT Rebindings
+
+{{Section 5.1.2 of QUIC-TRANSPORT}} indicates that an endpoint
+can change the Connection ID it uses for to another available one
+at any time during the connection. As such a sole change of the Connection
+ID without any change in the address does not indicate a path change and
+the endpoint will keep the same congestion control and RTT measurement state.
```

Comment:
Given that we give implementation considerations, we should not restrict the behaviour too much IMHO.

```suggestion
the endpoint can keep the same congestion control and RTT measurement state.
```